components/infobox/wikis/mobilelegends/infobox_league

```diff
@@ -0,0 +1,339 @@
+		end
+	end
+	return widgets
+end
+
+function League:addToLpdb(lpdbData, args)
+	lpdbData.participantsnumber = args.player_number or args.team_number
+
+	return lpdbData
+end
+
+function League:defineCustomPageVariables()
+	if _args.player_number then
+		Variables.varDefine('tournament_mode', 'solo')
+	end
+	Variables.varDefine('tournament_patch', _args.patch)
+	Variables.varDefine('tournament_endpatch', _args.epatch)
+
+	Variables.varDefine('tournament_publishertier', _args['moonton-sponsored'])
+
+	--Legacy Vars:
+	Variables.varDefine('tournament_edate', Variables.varDefault('tournament_enddate'))
+end
+
+function CustomLeague:_concatArgs(base)
+	local firstArg = _args[base] or _args[base .. '1']
+	if String.isEmpty(firstArg) then
+		return nil
+	end
+	local foundArgs = {mw.ext.TeamLiquidIntegration.resolve_redirect(firstArg)}
+	local index = 2
+	while not String.isEmpty(_args[base .. index]) do
+		table.insert(foundArgs,
+			mw.ext.TeamLiquidIntegration.resolve_redirect(_args[base .. index])
+		)
+		index = index + 1
+	end
+
+	return table.concat(foundArgs, ';')
+end
+
+function CustomLeague:_createPrizepool()
+	if String.isEmpty(_args.prizepool) and
+		String.isEmpty(_args.prizepoolusd) then
+		return nil
+	end
+
+	local localCurrency = _args.localcurrency
+	local prizePoolUSD = _args.prizepoolusd
+	local prizePool = _args.prizepool
+
+	prizePool = CustomLeague:_cleanPrizeValue(prizePool, localCurrency)
+	prizePoolUSD = CustomLeague:_cleanPrizeValue(prizePoolUSD)
+
+	if String.isEmpty(prizePool) and String.isEmpty(prizePoolUSD) then
+		return nil
+	end
+
+	if localCurrency then
+		localCurrency = string.upper(localCurrency)
+		local exchangeDate = Variables.varDefault('tournament_enddate', _TODAY)
+		local exchangeRate = CustomLeague:_currencyConversion(
+			1,
+			localCurrency,
+			exchangeDate
+		)
+		--set currency vars for usage in prize pools
+		Variables.varDefine('tournament_currency_rate', exchangeRate or '')
+		Variables.varDefine('tournament_currency', localCurrency)
+		if prizePool and not prizePoolUSD then
+			if not exchangeRate then
+				error('Invalid local currency "' .. localCurrency .. '"')
+			end
+			prizePoolUSD = exchangeRate * prizePool
+		end
+	end
+
+	Variables.varDefine('tournament_prizepoolusd', prizePoolUSD or prizePool)
+	Variables.varDefine('tournament_prizepoollocal', prizePool)
+
+	if prizePoolUSD and prizePool then
+		return Template.safeExpand(
+			mw.getCurrentFrame(),
+			'Local currency',
+			{(localCurrency or 'usd'):lower(), prizepool = CustomLeague:_displayPrizeValue(prizePool, 2)}
+		) .. '<br>(≃ $' .. CustomLeague:_displayPrizeValue(prizePoolUSD) .. ' ' .. _ABBR_USD .. ')'
+	elseif prizePoolUSD then
+		return '$' .. CustomLeague:_displayPrizeValue(prizePoolUSD, 2) .. ' ' .. _ABBR_USD
+	elseif prizePool then
+		return Template.safeExpand(
+			mw.getCurrentFrame(),
+			'Local currency',
+			{(localCurrency or 'usd'):lower(), prizepool = CustomLeague:_displayPrizeValue(prizePool, 2)}
+		)
+	end
+end
```

Comment:
Use Module:PrizePoolCurrency. See for example R6.

Reply:
Actually, should we maybe create a default handling on commons for this?

Reply:
Default handling on commons that can allow this highlighted section to be removed entirely to make way for commons would be great